script/build/repository.py

```
@@ -60,6 +59,16 @@ def generate_content(data_dir: Path, url: str) -> str:
 """
     return rv
 
+def json_update(json_filename: Path) -> str:
+    """Return the value of json_filename's "update" key."""
+    if not json_filename.suffix.lower() == "*.json":
```

Ruff (0.12.2)

Use `json_filename.suffix.lower() != "*.json"` instead of `not json_filename.suffix.lower() == "*.json"`

Replace with `!=` operator

(SIM201)